### PR TITLE
Add traditional chinese (zh-tw) for README and RULE

### PR DESCRIPTION
**Pull request by Asoul**
Issue #804 have finished here

### CODE REVIEW

**Comment by Asoul** [comment]
Oops, I think I created a new issue

**Comment by Asoul** [comment]
I've translate the original `README.md` and `RULEs.md` to `docs/README-zhtw.md` and `docs/RULES-zhtw.md`. Thanks.

File: docs/RULES-zhtw.md [hunk 1]
```diff
@@ -0,0 +1,1387 @@
...
+
+  ```js
+    function my_function () { }    // ✗ avoid
+    function myFunction () { }     // ✓ ok
+
+    var my_var = 'hello'           // ✗ avoid
+    var myVar = 'hello'            // ✓ ok
+  ```
+
+* **不允許行尾空白。**
```
Comment (floating-cat) [inline docs/RULES-zhtw.md]: typo:
Trailing commas not allowed. （不允許行尾逗號）

Comment (Asoul) [inline docs/RULES-zhtw.md]: Oops, thanks for remind

**Comment by feross** [comment]
@Asoul Thanks for this translation. It's so great to see the `standard` docs being translated into so many languages!

I added you (@Asoul) as well as @falmar and @alexmoreno who did the Spanish and Portuguese translations as GitHub collaborators for `standard` so you can continue to edit the documentation as it changes, or if you spot errors. Thanks for being awesome!

@Flet Can you rebuild the site so this goes live?

**Comment by falmar** [comment]
Nice there is whole lot of changes. I'll get into it as soon as I can

**Comment by Flet** [comment]
Its up http://standardjs.com/readme-zhtw

**Pull request merged**